src/ufo.rs

```diff
@@ -66,11 +67,43 @@ impl Default for Ufo {
             groups: None,
             kerning: None,
             features: None,
+            data_request: Default::default(),
             __non_exhaustive: (),
         }
     }
 }
 
+/// Which UFO data do you want?
+#[derive(Clone, Debug, PartialEq)]
+pub struct UfoDataRequest {
+    pub layers: bool,
+    pub lib: bool,
+    pub groups: bool,
+    pub kerning: bool,
+    pub features: bool,
+    __non_exhaustive: (),
+}
+
+impl UfoDataRequest {
+    /// Convenience function to create a UfoDataRequest with all fields either true or false.
+    pub fn from_bool(b: bool) -> Self {
+        UfoDataRequest {
+            layers: b,
+            lib: b,
+            groups: b,
+            kerning: b,
+            features: b,
+            __non_exhaustive: (),
+        }
+    }
+}
```

Comment:
thinking about this purely in terms of API: instead of `from_bool` I might just have two fns, `all` or `none`? Alternatively I might consider a builder pattern, where you do like,

```rust
let request = UfoDataRequest::new().layers(true).features(true);
```

or similar.